test/e2e/skopeo.go

```diff
@@ -24,7 +24,19 @@ const (
 )
 
 func openshiftRegistryAuth(client operatorclient.ClientInterface, namespace string) (string, error) {
-	sa, err := client.KubernetesInterface().CoreV1().ServiceAccounts(namespace).Get(context.TODO(), BuilderServiceAccount, metav1.GetOptions{})
+
+	var sa *corev1.ServiceAccount
+	var err error
+
+	// wait for the builder service account to exist and contain image pull secrets
```

Comment:
I caught a `index our of bounds` error against the ImagePullSecrets, so I've added a `waitFor` to ensure we can a) get the service account b) that is contains at least 1 image pull secret.

Reply:
Nice catch - it doesn't look like we did any sort of length checking before and this looks reasonable to me.

Reply:
Yeah, that's what was expecting too. It's really weird =S